jams/nsconvert.py

```diff
@@ -159,7 +165,13 @@ def note_midi_to_hz(annotation):
     '''Convert a pitch_midi annotation to pitch_hz'''
 
     annotation.namespace = 'note_hz'
-    annotation.data.value = 440 * (2.0 ** ((annotation.data.value - 69.0)/12.0))
+    data = annotation.pop_data()
```

Comment:
i can neither articulate why nor propose something "better", but the semantics of this feel weird for some reason.